src/io/config/ExportConfiguration.cpp

```diff
@@ -14,7 +14,12 @@ ExportConfiguration::ExportConfiguration(xml::XMLTag &parent)
   XMLTag::Occurrence occ = XMLTag::OCCUR_ARBITRARY;
   {
     XMLTag tag(*this, VALUE_VTK, occ, TAG);
-    tag.setDocumentation("Exports meshes to VTK text files.");
+    tag.setDocumentation("Exports meshes to VTK legacy format files. Parallel participants will use the VTU exporter instead.");
+    tags.push_back(tag);
+  }
+  {
+    XMLTag tag(*this, VALUE_VTU, occ, TAG);
+    tag.setDocumentation("Exports meshes to VTU files in serial or PVTU files with VTU piece files in parallel.");
```

Comment:
@fsimonis I just tried this functionality and would like to add the following:
Using the `vtu` exporter in serial does not create a `pvtu` file, but appends the rank number in the `vtu` file. Therefore, it is not possible to select the time-series in paraView, but only each file individually (the rank stays the same), which makes it somewhat useless. I would propose to 
1) either generate a `pvtu` file in serial runs as well 
2) drop the rank index in serial runs so that a time-series can still be visualized
3) use the serial vtk exporter even if a vtu exporter was configured.

Reply:
Thanks for pointing that out! A fix is coming in #1137